client/packages/invoices/src/Prescriptions/DetailView/PrescriptionLineEdit/PrescriptionLineEditPage.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useMemo } from 'react';
+import {
+  BasicSpinner,
+  DetailContainer,
+  ModalMode,
+  NothingHere,
+  RouteBuilder,
+  useBreadcrumbs,
+  useParams,
+} from '@openmsupply-client/common';
+
+import { ItemRowFragment, ListItems } from '@openmsupply-client/system';
+import { AppRoute } from '@openmsupply-client/config';
+import { PageLayout } from './PageLayout';
+import { usePrescription } from '../../api';
+import { AppBarButtons } from './AppBarButtons';
+import { PrescriptionLineEdit } from './PrescriptionLineEdit';
+
+export const PrescriptionLineEditPage = () => {
+  const { invoiceNumber, itemId } = useParams();
+  const { setCustomBreadcrumbs } = useBreadcrumbs();
+
+  const {
+    query: { data, loading: isLoading },
+  } = usePrescription();
+
+  const lines =
+    data?.lines.nodes.sort((a, b) => a.item.name.localeCompare(b.item.name)) ??
+    [];
+
+  let currentItem = lines.find(line => line.item.id === itemId)?.item;
+
+  let items = useMemo(() => {
```

Comment:
Not sure if useMemo is strictly needed here, but have had problems with things change on re-render before so was thinking this might helpful...